content/en/docs/reference/command-line-tools-reference/feature-gates/KubeletPodResourcesDynamicResources.md

```diff
@@ -8,10 +8,11 @@ _build:
 stages:
   - stage: alpha
     defaultValue: false
-    fromVersion: "1.27"  
+    fromVersion: "1.27"
+    toVersion: "1.33" 
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.34"
 ---
-Extend the kubelet's pod resources gRPC endpoint to
-to include resources allocated in `ResourceClaims` via `DynamicResourceAllocation` API.
-See [resource allocation reporting](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources) for more details.
-with information about the allocatable resources, enabling clients to properly
-track the free compute resources on a node.
+Extend the kubelet's pod resources gRPC List and Get endpoints to include resources allocated in
+ResourceClaims via [Dynamic Resource Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/).
```

Comment:
maybe link to "observability" section? Also, there is very little I can find on how DRA podresources API different from DevicePlugin. At least info on how devices are represented in podresources would be helpful

Reply:
fair point, adding